catalyst-voices-backend/Cargo.toml

```diff
@@ -1,8 +1,33 @@
-[package]
-name = "catalyst-voices-backend"
-version = "0.1.0"
+[workspace]
+resolver = "2"
+members = [
+    "src/event-db"
+]
+
+[workspace.package]
 edition = "2021"
+version = "0.0.1"
+authors = [ ]
+
+[workspace.dependencies]
+event-db = { path = "src/event-db" }
+
```

Comment:
Are there any reasons why we need to use empty lines as separators?

Reply:
I have added empty lines, to logically group dependecies based on their usage. Particulary these group will represent our own workspace crates.
I am seeing it as a convenient way how to distinguish them